examples/6tisch/simple-node/Makefile

```diff
@@ -19,6 +23,23 @@ MODULES += $(CONTIKI_NG_SERVICES_DIR)/shell
 
 ifeq ($(MAKE_WITH_ORCHESTRA),1)
 MODULES += $(CONTIKI_NG_SERVICES_DIR)/orchestra
+
+ifeq ($(MAKE_WITH_LINK_BASED_ORCHESTRA),1)
+ifeq ($(MAKE_WITH_STORING_ROUTING),1)
+CFLAGS += -DORCHESTRA_CONF_RULES="{&eb_per_time_source,&unicast_per_neighbor_link_based,&default_common}"
+else
+$(error "Inconsistent configuration")
+endif # MAKE_WITH_LINK_BASED_ORCHESTRA
+endif # MAKE_WITH_STORING_ROUTING
+
+endif # MAKE_WITH_ORCHESTRA
+
+ifeq ($(MAKE_WITH_STORING_ROUTING),1)
+MAKE_ROUTING = MAKE_ROUTING_RPL_CLASSIC
+CFLAGS += -DRPL_CONF_WITH_STORING=1
```

Comment:
This might be a mis-understanding: this flag only *includes* storing, does not have the root *enable* it.
That being said, it is enabled by default in Classic.
`RPL_CONF_MOP` is what tells to root whether to build a storing or non-storing network.

Reply:
Not sure if there is an action required?

Reply:
I would say configure `RPL_CONF_MOP` instead